pilot/pkg/xds/delta.go

```diff
@@ -390,6 +390,10 @@ func shouldRespondDelta(con *Connection, request *discovery.DeltaDiscoveryReques
 		}
 
 		res, wildcard, _ := deltaWatchedResources(nil, request)
+		skip := request.TypeUrl == v3.AddressType && wildcard
```

Comment:
So we skip if it's WDS and the initial xDS request (i.e. wildcard)? Maybe add a comment for why we don't need these resource names (especially for delta)